### PR TITLE
Only allow secondary profiles under 20 to join school network

### PR DESCRIPTION
**Pull request by the-kenneth**
Roughly estimate age to limit which accounts can join a school network

### CODE REVIEW

File: src/features/multi-profile/edit-profile/EditProfileScreen.tsx [hunk 1]
```diff
@@ -53,10 +53,11 @@ export const EditProfileScreen: React.FC<RenderProps> = (props) => {
         )}
 
         {editProfileCoordinator.shouldShowSchoolNetwork() && (
-          <>
```
Comment (andyguest) [inline src/features/multi-profile/edit-profile/EditProfileScreen.tsx:56]: what was this line?

File: src/features/multi-profile/edit-profile/EditProfileCoordinator.ts [hunk 2]
```diff
@@ -98,6 +98,19 @@ export class EditProfileCoordinator extends Coordinator implements UpdatePatient
...
   shouldShowSchoolNetwork() {
+    const currentPatient = this.patientData.patientState;
+    const birthYear = this.patientData.patientInfo?.year_of_birth;
+
+    // If undefined - the patient hasn't completed the register flow
+    if (!birthYear) {
+      return false;
+    }
+    const age = new Date().getFullYear() - birthYear;
+    const schoolAged = age > 1 && age < 20;
```
Comment (andyguest) [inline src/features/multi-profile/edit-profile/EditProfileCoordinator.ts:109]: Harsh on the kids of today! :-)

**Review by andyguest — APPROVED** [review]
This looks highly promising. Might be best to get a second opinion though :-)

File: src/features/multi-profile/edit-profile/EditProfileScreen.tsx (hunk 1 above)
Comment (selvach) [reply src/features/multi-profile/edit-profile/EditProfileScreen.tsx:56]: This is an empty View element - which is required to contain multiple `LinkItem` children.

**Pull request merged**